lib/pavilion/plugins/commands/status.py

```diff
@@ -37,70 +37,41 @@ def _setup_arguments(self, parser):
             '-s', '--summary', default=False, action='store_true',
             help='Display a single line summary of test statuses.'
         )
+        parser.add_argument(
+            '--history', default=False, action='store_true',
+            help='Display status history for a single test_run.'
+        )
```

Comment:
Create a 'mutually exclusive argument group for --summary and --history, since they are mutually exclusive.